sklearn/model_selection/_validation.py

```diff
@@ -871,7 +871,20 @@ def _index_param_value(X, v, indices):
 def permutation_test_score(estimator, X, y, groups=None, cv=None,
                            n_permutations=100, n_jobs=1, random_state=0,
                            verbose=0, scoring=None):
-    """Evaluate the significance of a cross-validated score with permutations
+    """Evaluate the significance of a cross-validated score by permuting
```

Comment:
Following pep8, we try to keep the first sentence to 1 line.

Reply:
Done in PR #14757.